packages/paste-design-tokens/tokens/global/background-color.yml

```diff
@@ -27,6 +27,9 @@ props:
     comment: Background color of the main page body
 
   # status
+  color-background-neutral-lightest:
+    value: "{!palette-blue-10}"
+    comment: Lightest neutral background color
```

Comment:
Should we be more explicit about these tokens, saying that they're used in Alerts and that they are paired with certain other colors?

Reply:
We should, and we have a ticket in the backlog to look at token meta data and pairing information. I'd punt to making these more desccriptive and detailed then so we can decide what we are expecting for all tokens